### PR TITLE
Describe `color_sync` device_option

### PR DESCRIPTION
**Pull request by sjorge**

### CODE REVIEW

File: docs/information/configuration.md [hunk 1]
```diff
@@ -280,6 +280,9 @@ The `configuration.yaml` allows to set device specific configuration. This can a
 * `filtered_attributes`: Allows to prevent certain attributes from being published. When a device would e.g. publish `{"temperature": 10, "battery": 20}` and you set `filtered_attributes: ["battery"]` it will publish `{"temperature": 10}`.
 * `optimistic`: Publish optimistic state after set, e.g. when a brightness change command succeeds Zigbee2MQTT assumes the brightness of the device changed and will publish this (default `true`).
 
+### Lights
+* `color_sync`: when set to true, all attributes in the MQTT color payload get syncronized from the active color_mode. (default `true`).
```
Comment (Koenkk) [inline docs/information/configuration.md:284]: I think it's better to document this on the device page itself (together with all other options). I will add this when I prepare the next release.

Comment (sjorge) [reply docs/information/configuration.md:284]: So have it listed for each light that supports either color_temp or color? That would probably be better yes.

Comment (Koenkk) [reply docs/information/configuration.md:284]: Yes that was my plan

Comment (sjorge) [reply docs/information/configuration.md:284]: SGTM, I'll close this one.

**Pull request closed without merging**